seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-2/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcOceanBaseMilvusIT.java

```diff
@@ -0,0 +1,437 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.connectors.seatunnel.jdbc;
+
+import org.apache.seatunnel.common.exception.SeaTunnelRuntimeException;
+import org.apache.seatunnel.common.utils.ExceptionUtils;
+import org.apache.seatunnel.e2e.common.TestResource;
+import org.apache.seatunnel.e2e.common.TestSuiteBase;
+import org.apache.seatunnel.e2e.common.container.ContainerExtendedFactory;
+import org.apache.seatunnel.e2e.common.container.EngineType;
+import org.apache.seatunnel.e2e.common.container.TestContainer;
+import org.apache.seatunnel.e2e.common.junit.DisabledOnContainer;
+import org.apache.seatunnel.e2e.common.junit.TestContainerExtension;
+
+import org.apache.commons.lang3.StringUtils;
+
+import org.awaitility.Awaitility;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.TestTemplate;
+import org.testcontainers.containers.Container;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.images.PullPolicy;
+import org.testcontainers.lifecycle.Startables;
+import org.testcontainers.milvus.MilvusContainer;
+import org.testcontainers.utility.DockerLoggerFactory;
+
+import com.google.common.collect.Lists;
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import io.milvus.client.MilvusServiceClient;
+import io.milvus.grpc.DataType;
+import io.milvus.grpc.MutationResult;
+import io.milvus.param.ConnectParam;
+import io.milvus.param.IndexType;
+import io.milvus.param.MetricType;
+import io.milvus.param.R;
+import io.milvus.param.RpcStatus;
+import io.milvus.param.collection.CreateCollectionParam;
+import io.milvus.param.collection.FieldType;
+import io.milvus.param.collection.LoadCollectionParam;
+import io.milvus.param.dml.InsertParam;
+import io.milvus.param.index.CreateIndexParam;
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.Driver;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Stream;
+
+import static org.awaitility.Awaitility.given;
+
+@Slf4j
+@DisabledOnContainer(
+        value = {},
+        type = {EngineType.SPARK, EngineType.FLINK},
+        disabledReason = "Currently SPARK and FLINK not support adapt")
+@Disabled("oceanbase vector and milvus takes up too much memory")
```

Comment:
Please remove disabled on fake to oceanbase case.

Reply:
At present, Oceanbase's vector image will not be closed, resulting in a process still running, and this image problem will be dealt with in the future, but the function is okay

Reply:
> Oceanbase's vector image will not be closed, resulting in a process still running

Do you know the reason? And is there any problem if we don't solve this problem and enable the test case at the same time?

Reply:
> Please remove disabled on fake to oceanbase case.

A PR will be submitted, the OB-related testcontainers will be modified, and the test container will be replaced to the latest version

Reply:
> > Oceanbase's vector image will not be closed, resulting in a process still running
> 
> Do you know the reason? And is there any problem if we don't solve this problem and enable the test case at the same time?

The problem of the ob vector container, I am sure that there is no problem with the test case, and the synchronous results can be printed out at the moment, but the test container still has threads running and cannot be closed

Reply:
ok.

Reply:
> At present, Oceanbase's vector image will not be closed, resulting in a process still running, and this image problem will be dealt with in the future, but the function is okay。seatunne's executeJob logic has an assertion to determine whether there is an image running, and after testing, replacing it with a mysql image or other versions of the image will not have this error, and the current ob vector image has this problem, so add @disable without modifying the original test logic on the premise。 ![image](https://private-user-images.githubusercontent.com/93303124/359763684-e82a89b4-a000-44d7-9084-9cfbbc51000c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQyMjAzNzEsIm5iZiI6MTcyNDIyMDA3MSwicGF0aCI6Ii85MzMwMzEyNC8zNTk3NjM2ODQtZTgyYTg5YjQtYTAwMC00NGQ3LTkwODQtOWNmYmJjNTEwMDBjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODIxVDA2MDExMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTEyY2Y0ZDljNDgxYWNhZWRiNjAyZGMxNjAxZmRjNDJhMjFmNDUzOTE0MjMwOGI0MDc5YzEwOTVlOWU0MGZmMWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Qlhp9eF2RNr8pWqPtVJJRjL3RFf8AhYGGqVi43cKMbE)

This is the assert logic provided by seatunnel to ensure that there are no abnormal threads in the engine. I believe this test case did not pass our assert logic. The `JNA cleaner` thread come from oceanbase client. This is a bug come from jna tool. Fixed by https://github.com/java-native-access/jna/commit/e8182b28e8baaaeae7b68767bca91c1012e39d4d in version [5.14.0](https://github.com/java-native-access/jna/releases/tag/5.14.0)
So we should do this:
1. open test case and add `JNA cleaner` thread into https://github.com/apache/seatunnel/blob/570bbb39a0de9deed60e4089823b80bc59ec3263/seatunnel-e2e/seatunnel-e2e-common/src/test/java/org/apache/seatunnel/e2e/common/container/seatunnel/SeaTunnelContainer.java#L407
2. waiting oceanbase client update jna version to 5.14.0 then remove it from `isIssueWeAlreadyKnow`